composer.json

```diff
@@ -52,8 +52,8 @@
     "scripts": {
         "post-root-package-install": [
             "php -r \"copy('.env.example', '.env');\"",
-            "php -r \"copy('app/config/development.example.php', 'app/config/development.php);\"",
-            "php -r \"copy('app/config/config.example.php', 'app/config/config.php);\""
+            "php -r \"copy('app/config/development.example.php', 'app/config/development.php');\"",
+            "php -r \"copy('app/config/config.example.php', 'app/config/config.php');\""
```

Comment:
We don't add changes directly to stable branches or `master`. Could you please send this PR to the `3.0.x` branch?